Src/Library/Messaging/Jobs/IJobStorageRecord.cs

```diff
@@ -28,4 +30,10 @@ public interface IJobStorageRecord
     /// indicates whether the job has successfully completed or not.
     /// </summary>
     bool IsComplete { get; set; }
+
+    /// <summary>
+    /// allows implementers to customize command parsing.
+    /// </summary>
+    [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
This attribute is not needed, compilers are very smart, just to express my intention of not disrupting performance even by a function call.